packages/flutter/test/material/slider_test.dart

```diff
@@ -4251,4 +4251,34 @@ void main() {
       );
     });
   });
+
+  testWidgets('Slider.onChanged is called only once', (WidgetTester tester) async {
+    int onChangeCallbackCount = 0;
+    late final double screenWidth;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: LayoutBuilder(builder: (BuildContext context, BoxConstraints constraints) {
+            screenWidth = constraints.maxWidth;
+            return Center(
+              child: Slider(
+                max: 5,
+                divisions: 5,
+                value: 0,
+                onChanged: (double newValue) {
+                  onChangeCallbackCount++;
+                },
+              ),
+            );
+          }),
+        ),
+      ),
+    );
```

Comment:
```suggestion
    await tester.pumpWidget(
      MaterialApp(
        home: Scaffold(
          body: Center(
            child: Slider(
              max: 5,
              divisions: 5,
              value: 0,
              onChanged: (double newValue) {
                onChangeCallbackCount++;
              },
            ),
          ),
        ),
      ),
    );

```

Reply:
We can omit `LayoutBuilder` layout and add a new line at the bottom. between this and `gesture` line.